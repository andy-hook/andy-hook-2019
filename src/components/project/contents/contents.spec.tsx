import React from "react"
import { projectsData } from "@mock-data"

import Contents from "./contents"
import { shallowWithTheme } from "@test-utils"

describe("<Contents />", () => {
  it("renders correctly", () => {
    const tree = shallowWithTheme(
      "dark",
      <Contents sections={projectsData.brandwatch.contents} />
    )
    expect(tree).toMatchSnapshot()
  })
})
