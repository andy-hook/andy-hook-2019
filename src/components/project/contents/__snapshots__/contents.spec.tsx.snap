// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contents /> renders correctly 1`] = `ShallowWrapper {}`;
